operator_cli/commands/sync_local.py

```diff
@@ -0,0 +1,54 @@
+import click
+from eth2deposit.settings import MAINNET
+from requests.exceptions import ConnectionError, HTTPError
+from web3.beacon import Beacon
+
+from operator_cli.eth2 import validate_mnemonic
+from operator_cli.local_storage import LocalStorage
+from operator_cli.settings import SUPPORTED_CHAINS
+
+
+def get_beacon_client() -> Beacon:
+    url = click.prompt("Please enter the ETH2 node URL", type=click.STRING)
+    return Beacon(base_url=url)
+
+
+@click.command(help="Synchronizes validator keystores in the local storage")
+@click.option(
+    "--chain",
+    default=MAINNET,
+    help="The network of ETH2 you are targeting.",
+    prompt="Choose the (mainnet or testnet) network/chain name",
+    type=click.Choice(SUPPORTED_CHAINS.keys(), case_sensitive=False),
+)
+def sync_local(chain: str) -> None:
+    while True:
+        try:
+            beacon_client = get_beacon_client()
+            beacon_client.get_genesis()
+            break
+        except (ConnectionError, HTTPError):
+            pass
+
+        click.echo("Error: failed to connect to the ETH2 server with provided URL")
+
+    mnemonic = click.prompt(
```

Comment:
Would be also great to ask for the directory where to store the keys. The default could be `validator_keys` in the same dir.